modules/wlogout/hm.nix

```
@@ -0,0 +1,168 @@
+{
+  mkTarget,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  iconNames = [
+    "lock"
+    "logout"
+    "suspend"
+    "hibernate"
+    "shutdown"
+    "reboot"
+  ];
+  mkIcons =
+    color:
+    pkgs.runCommand "stylix-wlogout-icons" { buildInputs = [ pkgs.imagemagick ]; }
+      ''
+        ICONS=(${lib.concatStringsSep " " iconNames})
+        mkdir -p $out
+        for icon in "''${ICONS[@]}"; do
+          sed ${pkgs.wlogout.src}/assets/$icon.svg \
+            -e "s/<svg/<svg fill=\"${color}\"/" \
+            | convert -background none - -resize 512x512 $out/$icon.png
+        done
+      '';
+in
+mkTarget {
+  name = "wlogout";
+  humanName = "wlogout";
+
+  extraOptions =
+    { colors }:
+    {
+      background = lib.mkOption {
+        type = lib.types.nullOr lib.types.str;
+        description = "Used to set bg even if `opacity` or `colors` is null";
+        internal = true;
+        default = null;
+      };
+      font = lib.mkOption {
+        type = lib.types.enum [
+          "serif"
+          "sansSerif"
+          "monospace"
+          "emoji"
+        ];
+        default = "sansSerif";
+        example = "monospace";
+        description = "The stylix font for wlogout to use";
+      };
+      addCss = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        example = false;
+        description = "Adds fully functional css (otherwise just adds colors and fonts)";
+      };
+      iconColor = lib.mkOption {
+        type = with lib.types; nullOr str;
+        default = colors.withHashtag.base0E or null;
+        example = "#ff0000";
```

What about matching the `defaultText` expression:

```suggestion
        example = colors.withHashtag.base0E;
```